### DISCUSSION TITLE
Support for URP 17 Render Graph System

### DISCUSSION THREAD

**Discussion by NAC-42** [happy-turtle/oit-unity#41] [General]
Hi there, awesome work in this project!
Was wondering if there are any plans for this to be factored in to the render graph system for URP 17?

**Comment by happy-turtle** [comment]
Hey, I gave the new API a look and yeah sure, it sounds nice and more stability would also be nice. We could definitely try it out and move the URP implementation to the new API once Unity 6 is released.

As far as I understand that would mean though that the package would require URP 17 and Unity 6 then, without backwards compatibility.

**Reply by NAC-42** [reply]
That's awesome news, thanks for getting back to me!

Yes I do believe that would be the case, at the moment there is a compatibility mode which would allow the package to keep the majority of it's structure in place, although I also believe that mode is merely in place for transitioning to the newer system. It looks to be something that they may remove in later versions past release.

Looking forward to how it goes for you!
